rpc/gnmi/gnmi-path-conventions.md

```diff
@@ -1,71 +1,146 @@
 ## Schema path encoding conventions for gNMI
 
-**Updated**: October 13, 2016  
-**Version**: 0.1.0
+**Updated**: February 24, 2016<br>
+**Version**: 0.2.0
 
-This document is a supplement to the [gNMI specification](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md), and provides additional guidelines and examples for path encoding.
+This document is a supplement to the [gNMI
+specification](https://github.com/openconfig/reference/blob/master/rpc/gnmi
+/gnmi-specification.md), and provides additional guidelines and examples for
+path encoding.
 
 ### Path encoding
 
-Data model path encoding and decoding is required in a gNMI-compliant management stack.  For streaming telemetry, devices (targets) generate notifications, or updates, with an associated path and value.  The data collector must interpret these paths efficiently.  For device configuration, the NMS client generates configuration data with an associated path, and passes the data to the target, which must correctly interpret the path against the appropriate data tree and apply the configuration.
+Data model path encoding and decoding is required in a gNMI-compliant
+management stack.  For streaming telemetry, devices (targets) generate
+notifications, or updates, with an associated path and value.  The
+data collector must interpret these paths efficiently.  For device
+configuration, the NMS client generates configuration data with an
+associated path, and passes the data to the target, which must
+correctly interpret the path against the appropriate data tree and
+apply the configuration.
 
 ### Guidance for implementors
 
-Path encoding in gNMI uses a variant of XPATH syntax with the following rules.  Some of the examples below assume YANG-modeled data since it is a common use case for gNMI, but gNMI has applicability to any data modeling where the data can be described in a tree structure with hierarchical paths.
-
-
+Path encoding in gNMI uses a variant of XPATH syntax with the
+following rules.  Some of the examples below assume YANG-modeled data
+since it is a common use case for gNMI, but gNMI has applicability to
+any data modeling where the data can be described in a tree structure
+with hierarchical paths.
+
+#### Constructing paths
+
+gNMI paths are encoded as a list (slice, array etc.) of path string elements.
+
+*  the root path `/` is encoded as a zero length array (slice) of path
+   elements.  Example declarations in several languages:
+```
```

Comment:
Perhaps specify which language is being used here for each example in a bulleted list.